main.go

```diff
@@ -961,6 +963,22 @@ func getGlobalStorageHandler(keyPrefix string, hashKeys bool) storage.Handler {
 
 func main() {
 	arguments := getCmdArguments()
+	if arguments["lint"] == true {
+		path, lines, err := lint.Run(confPaths)
+		if err != nil {
+			fmt.Fprintln(os.Stderr, err)
+			os.Exit(1)
+		}
+		if len(lines) == 0 {
+			fmt.Printf("found no issues in %s\n", path)
+			return
+		}
+		fmt.Printf("issues found in %s:\n", path)
+		for _, line := range lines {
+			fmt.Println(line)
+		}
+		return
```

Comment:
Shouldn't this return a non-zero exit code too?

Reply:
Ah, very well seen.

Reply:
done - it's `1` now.